definitions/ext-firewall/definition.yml

```diff
@@ -0,0 +1,30 @@
+domain: EXT
+# Firewall devices from Kentik using Metric API
+# Note this initial definition is limited to only Fortigate Firewalls using the FORTINET-FORTIGATE-MIB
+# Future firewall entities will need to be isolated in the definition
+type: FIREWALL
+synthesis:
+  name: device_name
+  identifier: device_name
+  encodeIdentifierInGUID: true
+
+  conditions:
+  - attribute: provider
+    value: kentik-firewall
+
+  tags:
+    src_addr:
+      entityTagName: device_ip
+
+compositeMetrics:
+  goldenMetrics:
+  - golden_metrics.yml
+  summaryMetrics:
+  - summary_metrics.yml
+
+goldenTags:
+- device_ip
+
+dashboardTemplates:
+  newRelic:
```

Comment:
You could use `kentik` here instead (that's the value of the `instrumentation.provider` for this telemetry right?) that way it'd be easier for people to extend this definition for other providers

Reply:
that's a good callout - I was thinking to wait until the new standard that @srvaroa and folks are working on is in place for using `instrumentation.provider` ++ ??? and then I could push an update to all of the Kentik entities at once. 

Would you rather I submit the change to `kentik` on this PR and then update later?

Reply:
Yeah I think adding here kentik would be less confusing in the future.

When we adde those changes we still will need to do some batch updates here anyway.

Reply:
change added; thanks!

Reply:
thanks!